package.json

```diff
@@ -5,18 +5,9 @@
   "registry": "brigadecore",
   "tag": "latest",
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
```

Comment:
I would very much like to keep the commands in `package.json` as well, as this is the default way Angular developers work.

Reply:
I put them back. I also added a make target that will launch the development server in a container.